src/index.js

```diff
@@ -43,6 +44,10 @@ const snapshotDiff = (valueA: any, valueB: any, options?: Options): string => {
     difference = stripAnsi(difference);
   }
 
+  if (mergedOptions.omitPatchMarks) {
+    difference = difference.replace(/^@.*/gm, '-------------')
```

Comment:
I think we can make this regex better and take into account that we have two @@ and numbers with commas between. Also I'm leaning towards replacing them with:
`@@ --- --- @@`

Reply:
Also we only need to check it only if expand === false

Reply:
👍

Reply:
I considered a more rigorous regex, but since `jest-diff` seems to always pad the start of the line with whitespace, I went the lazy route. But I agree this should be improved. Here's an example of the new regex: https://regex101.com/r/KLDkC0/1

Reply:
Ha, left out the 0s in the number range. Updated regex: https://regex101.com/r/KLDkC0/2